src/meta/model_v2/migration/src/m20240525_090457_secret.rs

```diff
@@ -37,12 +37,22 @@ impl MigrationTrait for Migration {
             )
             .await?;
 
-        // Add a new column to the table
+        // Add a new column to the `sink` table
         manager
             .alter_table(
                 MigrationTable::alter()
                     .table(Sink::Table)
-                    .add_column(ColumnDef::new(Sink::SecretRef).json_binary())
+                    .add_column(ColumnDef::new(Sink::SecretRef).blob(BlobSize::Long))
```

Comment:
```suggestion
                    .add_column(ColumnDef::new(Sink::SecretRef).binary())
```
I guess this column won't be that large, `binary` is enough.